src/serving/route_prediction.py

```diff
@@ -57,7 +57,11 @@ def _get_gas_station_points_near_path(path, radius=0.02):
 def predict_price(id, time):
     model, _, df_forecast = train_and_predict(gas_station_id=id, start_time=time, end_time=time, use_cached=True,
                                               cache=True)
-    deci_cent = round(Decimal(df_forecast.loc[0, 'yhat']), 0)
+    if df_forecast is None:
```

Comment:
Is this really a clean solution? Wouldn't e.g. -1 express better that the prediction we give is wrong?